README.md

```diff
@@ -218,7 +218,7 @@ The code is set up as an ordinary leiningen project, with the one exception
 that you need to run `make` before running `lein test` or `lein run`, as
 there are messages that need to be turned into a message bundle.
 
-# Maintenance
+# Maintaineance
```

Comment:
Wow this is weird, why does this keep getting changed to Maintaineance.  Can you fix the spelling on that :)  Thanks JANE!!!